translations/locale_de.ts

```diff
@@ -33,7 +33,8 @@ Siehe auch: https://musescore.org/en/handbook/3/tempo#ritardando-accelerando</tr
 TempoChanges needs v3.0.5 or above.
 </source>
         <translation>Nicht unterstützte MuseScore Version.
-TempoChanges benötigt v3.0.5 oder höher.</translation>
+TempoChanges benötigt v3.0.5 oder höher.
+</translation>
```

Comment:
Do all Germans need more vertical space, or just you? ;-)

Reply:
It is the Belgian Dutch actually ;-) the original text ends with a linefeed, I removed that in the German translation earlier, but don't anymore, Qt Linguist complained about a difference in whitespace

Reply:
Oh my, stupid typo on my part.
Then again, laziness tells me it won't hurt anybody. I'll only update that if any other text happens to change in the future.

Given the small size of the project I've just filled out translations with Notepad++.